content/en/docs/develop/core-modules/bank.md

```diff
@@ -6,23 +6,94 @@ weight: 40
 {{< alert >}}
 **Note**
 
-XPLA Chain's bank module inherits from the Cosmos SDK's [`bank`](https://docs.cosmos.network/master/modules/bank/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
+XPLA Chain's bank module inherits from the Cosmos SDK's [`bank`](https://docs.cosmos.network/v0.45/modules/bank/) module. This document is a stub and covers mainly important XPLA Chain-specific notes about how it is used.
 {{< /alert >}}
 
 The bank module is the base transactional layer of the XPLA Chain. This module allows assets to be sent from one `Account` to another. The bank module defines the following types of send transactions: `MsgSend` and `MsgMultiSend`.
 
+The bank module is responsible for handling multi-asset coin transfers between accounts and tracking special-case pseudo-transfers which must work differently with particular kinds of accounts (notably delegating/undelegating for vesting accounts). It exposes several interfaces with varying capabilities for secure interaction with other modules which must alter user balances.
+
+In addition, the bank module tracks and provides query support for the total supply of all assets used in the application.
+
+This module will be used in the Cosmos Hub.
```

Comment:
plz update the network name

Reply:
https://github.com/xpladev/docs/pull/13/commits/e7d6947e618aa9f40bb0ee16fc87d21b2e9434fe